### DISCUSSION TITLE
Plugins for cross line feature?

### DISCUSSION THREAD

**Discussion by narsireddypanta** [Leaflet/Leaflet#7881] [General]
Hi Team,

Is there any plugin that support fixed cross lines on map like below

<img width="687" alt="Screenshot 2021-12-30 at 4 37 29 PM" src="https://user-images.githubusercontent.com/68415004/147746667-ae027f6f-6773-4c5a-8c3d-87eab71b14b4.png">

**Comment by IvanSanchez** [comment]
A crosshair? Do see e.g. http://xguaita.github.io/Leaflet.MapCenterCoord/ (and the rest of https://leafletjs.com/plugins.html )

**Reply by narsireddypanta** [reply]
@IvanSanchez  Not cross hair type.. a fixed full cross lines on top of map. I checked all leaflet plugins but no luck

**Reply by jonkoops** [reply]
Perhaps by adding some DOM and CSS to the containing element for the map you could style this in user-space.

**Reply by narsireddypanta** [reply]
Ok.
